src/test/java/org/opensearch/security/test/helper/cluster/ClusterHelper.java

```diff
@@ -308,13 +308,12 @@ public ClusterInfo waitForCluster(final ClusterHealthStatus status, final TimeVa
 
             final List<NodeInfo> nodes = res.getNodes();
 
-            final List<NodeInfo> clusterManagerNodes = nodes.stream().filter(n->n.getNode().getRoles().contains(DiscoveryNodeRole.CLUSTER_MANAGER_ROLE)).collect(Collectors.toList());
+            final List<NodeInfo> clusterManagerNodes = nodes.stream().filter(n->n.getNode().getRoles().stream().anyMatch(r -> r.roleName().equals("master"))).collect(Collectors.toList());
```

Comment:
We should not be using the term master in our code anymore, it should be cluster manager or something similar